File: src/lib/luxorigin/LanguageAPI/translate/Translate.php
<?php

declare(strict_types=1);

namespace lib\luxorigin\LanguageAPI\translate;

readonly class Translate
{
    public function __construct(
        protected string $key,
        protected array $parameters = []
    ) {}

    public function getKey(): string
    {
        return $this->key;
    }

    /**
     * @return string[]
     */
    public function getParameters(): array
    {
        return $this->parameters;
    }
}

File: paper/src/main/java/net/lib/luxorigin/LanguageAPI/translate/Translate.java
package net.lib.luxorigin.LanguageAPI.translate;

import lombok.Getter;

import java.util.Map;

@Getter
public class Translate {

    private final String key;
    private final Map<String, String> translate;

    public Translate(String key, Map<String, String> translate) {
        this.key = key;
        this.translate = translate;
    }
}